packages/o/openmvg/xmake.lua

```diff
@@ -0,0 +1,103 @@
+package("openmvg")
+    set_homepage("https://github.com/openMVG/openMVG")
+    set_description("open Multiple View Geometry library. Basis for 3D computer vision and Structure from Motion.")
+    set_license("MPL-2.0")
+
+    add_urls("https://github.com/openMVG/openMVG/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/openMVG/openMVG.git")
+    add_versions("v2.1", "5538c1c8453385e5595047198fc9d0e3d8cdaa85a53b3a27c70cb05ae3f5fc2c")
+
+    add_configs("openmp", {description = "Enable OpenMP parallelization", default = true, type = "boolean"})
+    if is_plat("windows") then
+        add_configs("shared", {description = "Build shared library", default = false, type = "boolean", readonly = true})
+    end
+
+    add_deps("cmake")
+    if is_subhost("windows") then
+        add_deps("pkgconf")
+    else
+        add_deps("pkg-config")
+    end
+    add_deps("eigen", "libpng", "libjpeg", "libtiff", "flann", "lemon", "cereal", "ceres-solver", "coin-or-clp")
+
+    if on_check then
+        on_check("linux", function (package)
+            assert(not package:has_tool("cxx", "clang"), "Linux Clang is not supported.")
+        end)
+    end
+
+    on_load(function (package)
+        if package:config("openmp") then
+            package:add("deps", "openmp")
+        end
+
+        local libs = {"openMVG_easyexif", "openMVG_exif", "openMVG_fast", "openMVG_features", "openMVG_geometry", "openMVG_image", "openMVG_kvld", "openMVG_lInftyComputerVision", "openMVG_linearProgramming", "openMVG_matching", "openMVG_matching_image_collection", "openMVG_multiview", "openMVG_numeric", "openMVG_robust_estimation", "openMVG_sfm", "openMVG_stlplus", "openMVG_system", "vlsift"}
+        for _, lib in ipairs(libs) do
+            package:add("links", lib)
```

Comment:
you can `package:add("links", libs)` instead of for-loop

Reply:
and why not use `add_links("openMVG_easyexif", "openMVG_exif", "openMVG_fast", "openMVG_features", "openMVG_geometry", "openMVG_image", "openMVG_kvld", "openMVG_lInftyComputerVision", "openMVG_linearProgramming", "openMVG_matching", "openMVG_matching_image_collection", "openMVG_multiview", "openMVG_numeric", "openMVG_robust_estimation", "openMVG_sfm", "openMVG_stlplus", "openMVG_system", "vlsift")` in description scope.